### ISSUE TITLE
Add PATCH method into PermissionService

### ISSUE THREAD

**Issue by anthonybriand** [tjwebb/sails-permissions#61]
Please add support of the PATCH method in PermissionService

**Comment by tjwebb** [comment]
Can you be more specific as to what exactly it would do?

Contributions are welcome. I can't always implement all feature requests

**Comment by anthonybriand** [comment]
Yes, it would be fine to add the support of the HTTP PATCH method, this method is used for updating a chunk of a model record

eg : PATCH /controller/[id] with data ->
```
{
    name: 'Hello'
}
```
Will only update the "name" attribute of the model [id], this method works like a PUT but contain only modified attributes
